dockerfiles/base/all-tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Now, when we decided to do it other way, this all tests "suite" can be removed and test execution commands can be added to build.sh. I intended to include all-tests.sh execution in build.sh. To have all-tests.sh a place to track .bats files to execute and to have just plain shell script calling in build.sh.